### DISCUSSION TITLE
Offline Redirect URL for Room links

### DISCUSSION THREAD

**Discussion by MuhammadSalal** [mozilla/hubs#4419] [Hubs Cloud Questions] [answered]
Hi guys, 
I had a question regarding redirecting URL 
for not only the stack formation once it is offline, but also the room links. 
Is there a way to have them both redirected to the same domain name? Because it works fine if someone just types our domain name, but if our servers are offline and someone tries to access one of our rooms via link, then it gives them a 404 error.
If there is way to redirect all the room links to that domain name as well then that would be great. 

<img width="502" alt="URL Redirect" src="https://user-images.githubusercontent.com/78027141/125900868-666c393c-5338-4a31-835e-d38067f57bf2.PNG">

**Comment by pattersonbl2** [comment, marked as answer]
Made an issue in Hubs-cloud repo: https://github.com/mozilla/hubs-cloud/issues/214
